example/spec/parallel/retryConfiguration/retryConfigurationSpec.js

```diff
@@ -118,7 +118,10 @@ describe('When a task is configured', () => {
   describe('with a backoff', () => {
     it('and it fails, Cumulus retries it with the configured backoff time', () => {
       const retryIntervals = getRetryIntervals(retryFailLambdaExecutions);
-      expect(retryIntervals).toEqual([2, 4, 8]);
+      const expectedIntervals = [2, 4, 8];
+      expectedIntervals.forEach((expected, idx) => {
+        expect(Math.abs(expected - retryIntervals[idx])).toBeLessThanOrEqual(1);
```

Comment:
Do we need to allow for intervals of less than 2,4,8? I thought we just needed to allow for intervals of 3,5,9 because of rounding up?

Reply:
I was just guessing that if it could round up incorrectly a fast start on one would cause the rest to also be fast.  I just switched to allow a second on either side of expected. Let me know if you only want later and I'll remove the Math.abs (I think)

Reply:
It's a nit, so I'll just approve

Reply:
It's not hard, I can fix it (just have to do two expects.). I'll do that when I finish my current PR